lib/Reader/abisignature.cpp

```diff
@@ -247,6 +247,20 @@ static Value *getFieldAddress(IRBuilder<> &Builder, Value *Base,
   return Address;
 }
 
+bool ABICallSignature::hasIndirectResultOrArg() const {
```

Comment:
Note that you'll need to avoid testing using P/Invokes that return structs by value until [D14465](http://reviews.llvm.org/D14665) is merged.